config/bundles.php

```diff
@@ -42,4 +42,5 @@
     FOS\JsRoutingBundle\FOSJsRoutingBundle::class => ['all' => true, 'admin' => true],
     Symfony\Cmf\Bundle\RoutingBundle\CmfRoutingBundle::class => ['all' => true, 'website' => true],
     Stof\DoctrineExtensionsBundle\StofDoctrineExtensionsBundle::class => ['all' => true],
+    Sulu\Bundle\EventLogBundle\SuluEventLogBundle::class => ['all' => true],
```

Comment:
last position?

Reply:
problem is that not the newest commit of sulu/sulu is installed